src/Operators.jl

```diff
@@ -67,15 +76,13 @@ sub(x, y) = x - y
 mult(x, y) = x * y
 pow_abs(x, y) = abs(x)^y
 div(x, y) = x / y
-log_abs(x) = log(abs(x) + 1//100000000)
-log2_abs(x) = log2(abs(x) + 1//100000000)
-log10_abs(x) = log10(abs(x) + 1//100000000)
-log1p_abs(x) = log(abs(x) + 1)
-acosh_abs(x) = acosh(abs(x) + 1)
+log_nan(x) = log(x)
```

Comment:
Could these be made equivalent to their non-generic versions - e.g., `(x <= 0) ? NaN : log(x)`?

Reply:
Not every `x` works in that context. If `x`, I presume, is `<: Number` it works, but for example in the context of `SymbolicUtils` `x` is not, and `x <= 0` cannot be evaluated. In other terms, if `x` is a symbolic expression, we do not know if `x` is smaller or equal to zero or not before it has been evaluated.

Reply:
Good point! Sounds good to me.